src/app/features/videomania/index.js

```
@@ -0,0 +1,85 @@
+      .addCase(getVideoCategories.fulfilled, (state, action) => {
+        state.isFetching = false;
+        state.categories = action?.payload;
+      })
+      .addCase(getVideoCategories.rejected, (state, action) => {
+        state.isFetching = false;
+        state.error = action?.payload;
+      })
+      .addCase(getVideosByCategory.pending, (state) => {
+        state.isFetching = true;
+      })
+      .addCase(getVideosByCategory.fulfilled, (state, action) => {
```

ESLint

[error] 'action' is defined but never used.

(no-unused-vars)